SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -638,6 +640,8 @@ void ActorSystem::EntityRemoved(const Worker_EntityId EntityId)
 {
 	SCOPE_CYCLE_COUNTER(STAT_ActorSystemRemoveEntity);
 
+	UnregisterStablePathEntity(EntityId);
```

Comment:
uber nit: move this till after the RemoveActor call to better match the EntityAdded flow.